content/docs/start/data-pipelines.md

```diff
@@ -217,30 +217,29 @@ Please check the `dvc.yaml` again, it should have one more stage now.
 
 </details>
 
-This should be a good point to commit the changes with Git. These include
+This should be a good time to commit the changes with Git. These include
 `.gitignore`, `dvc.lock`, and `dvc.yaml` — which describe our pipeline.
 
 ## Reproduce
 
-The whole point of creating this `dvc.yaml` pipelines file is an ability to
-reproduce the pipeline:
+The whole point of creating this `dvc.yaml` file is the ability to easily
```

Comment:
Actually the term "pipelines file" is a thing: https://dvc.org/doc/user-guide/project-structure/pipelines-files

Reply:
not really obvious (similar to "params file"). If these are terms they should be markedup in some way (italicise, underline, etc) so that it doesn't look like grammatically incorrect usage.

Reply:
Whata bout italics for now? *pipelines file*

I'm not sure it's grammatically incorrect (file with pipelines) but it does sounds strange.

Reply:
- it's not incorrect, it just sounds incorrect. Yes, it's a problem which probably has to be sorted out in all the docs. I'd vote hyphen (since that's the usual thing to do with compound terms) but italics etc is fine
- for this particular sentence though, let's not repeat the word multiple times. "Pipeline" already appears at the end of the sentence.

Reply:
I like the hyphenation idea, want to open an issue to confirm and apply globally?

> let's not repeat the word multiple times

I see, OK. Would be nice to have the term somewhere in this page though.